react/Header/UserAccount/UserAccount.js

```diff
@@ -107,11 +106,6 @@ export default class UserAccount extends Component {
         </div>
 
         <div className={styles.accountContainer}>
```

Comment:
This div was added in https://github.com/seek-oss/seek-style-guide/pull/621/files. Should we remove it in this PR? There are also some styling changes in #621 that have not been reversed. Any reason why not?

Reply:
Have removed `accountContainer` div. Styling changes were improvements that were made that don't need to be removed.